### ISSUE TITLE
NL-RSE logo

### ISSUE THREAD

**Issue by c-martinez** [nl-rse/nl-rse.github.io#55]
We should create our own logo, with a similar style to the other RSE groups, but specifically for our community and clearly identifiable as a separate organisation

**Comment by mkuzak** [comment]
Do you have anything specific in mind? More like de-rse or maybe us-rse logo? That something that should be discussed in the Core Team.

**Comment by c-martinez** [comment]
No specific ideas -- just something to discuss with the core team. Tom also said we can probably hire a designer for it.

**Comment by mkuzak** [comment]
Yes, definitely the desginer. We should not be designing anything ourselves.

**Comment by c-martinez** [comment]
Let's discuss it with the rest of the core group on our next meeting.
